src/devices/awox.ts

```diff
@@ -1,10 +1,84 @@
 import * as fz from "../converters/fromZigbee";
 import * as exposes from "../lib/exposes";
 import * as m from "../lib/modernExtend";
-import type {DefinitionWithExtend} from "../lib/types";
+import type {DefinitionWithExtend, Fz} from "../lib/types";
 
 const e = exposes.presets;
 
+const fzLocal = {
+    awoxRemoteActions: {
+        cluster: "genOnOff",
+        type: [
+            "commandOn",
+            "commandOff",
+            "raw",
+            "commandStepWithOnOff",
+            "commandStep",
+            "commandEnhancedMoveHue",
+            "commandRecall",
+            "commandStepColorTemp",
+        ],
+        convert: (model, msg, publish, options, meta) => {
+            const payload = msg.data;
+            let action = null;
+
+            if (msg.cluster === "lightingColorCtrl") {
```

Comment:
After checking this code more closely I think this solution you proposed makes more sense:

> Would you prefer if I tried to strip down awox_remote_actions to only handle the cases that are definitively not covered by fz.command_on, fz.command_off, fz.command_step, fz.command_move, etc., and then add awox_remote_actions in the fromZigbee list alongside the generic ones? This would make the definition cleaner.

e.g. the `else if (msg.type === "commandStepColorTemp") {` is just renamed from `color_temperature_move_up` to `color_temp_warm`, I don't think that deserves a breaking change and duplicate actions being published. So I guess I only makes sense to keep the ` if (msg.cluster === "lightingColorCtrl") {` branch.